docs/configuration.md

```diff
@@ -43,7 +43,7 @@ For example, the config object may look similar to the following:
 | telemetries | [Telemetry Config Array](#telemetry-config-array) | `[]` | See [Telemetry Config Array](#telemetry-config-array) |
 | batchLimit | Number | `100` | The maximum number of events that will be sent in one batch of RUM events. |
 | dispatchInterval | Number | `5000` | The frequency (in milliseconds) in which the webclient will dispatch a batch of RUM events. RUM events are first cached and then automatically dispatched at this set interval. |
-| eventCacheSize | Number | `200` | The maximum number of events the cache can contain before dropping events. |
+| eventCacheSize | Number | `1000` | The maximum number of events the cache can contain before dropping events. Each event is typically 1KB in size, so we recommend a default limit of 1000 events -> 1 MB to balance performance against capturing all observed events. If necessary, feel free to enable debug mode to get detailed logs on how to optimize cache size depending on your application behavior. |
```

Comment:
question: should we clarify the 1KB thing to be for non custom events?

Reply:
it's typically 1kb due to metadata, so honestly it'll probably be the same for custom events